### DISCUSSION TITLE
How to correctly specify the URL for the auto-open in container

### DISCUSSION THREAD

**Discussion by GeniusBroccoli** [mbnuqw/sidebery#906] [Q&A] [answered]
Hi. I can't figure out how to correctly specify the URL, tried both, just domain like google.com and using regex, the site does not open in the container. I use stable version of sidebery (@slrslr help :smiling_face_with_tear: ).

**Comment by slrslr** [comment, marked as answer]
Hello, I have Sidebery container set like this:
![sidebery-container-proxy](https://user-images.githubusercontent.com/6596726/213280190-41905be5-4a02-4eb7-82aa-e1754285acc9.gif)
Make sure you have all switches and values like me. Obviously different IP, port and consider proxy type SOCKS etc.
Then opening https://ipv6-test.com/ in a new tab of my Firefox opens tab as Personal/red highlight. Make sure to restart Firefox. NOT use private browsing window. Check that Sidebery plugin is enabled on about:addons

**Reply by GeniusBroccoli** [reply]
I got the containers working, but I didn't figured out what helped. Initially, with the extension just installed, I enabled the container and entered the domain, gave the requested permissions, but the site would not open in the container. Now, I turned on the proxy and user agent settings, then turned them off, and now the container works. Newly created containers also work. There's probably something turned on somewhere that didn't work initially, but I don't know what, or maybe I'm just dumb :tada: 
Anyway, it's working now, thanks for the help.

**Reply by Fred-Vatin** [reply]
Thanks. I had the same problem in v5 after I disabled the **Firefox Multi-account container** extension to only use sidebery. I switch proxy and user agent and now it works as intended. Sounds like a bug.
